docs/ENVIRONMENT_VARIABLES.md

```diff
@@ -20,18 +20,17 @@ If an environment variable is updated, make sure to restart the server for the a
-| `LOGIN_BASE_URL`                    | string  | ""                                                                                           | The base URL used to construct the environment-dependent login/logout link.                                                             |
-| `NEXT_PUBLIC_REVERSE_PROXY_ENABLED` | boolean | true                                                                                         | Feature flag that disables Next router navigation on Searches to fix issues navigating between research-catalog and discovery-front-end |
-| `SIERRA_BASE`                       | string  | ""                                                                                           | Sierra base url                                                                                                                         |
-| `SOURCE_EMAIL`                      | string  | ""                                                                                           | Default source email used in feedback form submissions                                                                                  |
-| `LIB_ANSWERS_EMAIL`                 | string  | ""                                                                                           | Destination email for feedback form submissions                                                                                         |
+| Variable                      | Type   | Value Example                                                                                | Description                                                                                                  |
+| ----------------------------- | ------ | -------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------ |
+| `NEXT_PUBLIC_APP_ENV`         | string | "development"                                                                                | App environment key used to determine various environment-specific app settings.                             |
+| `NYPL_HEADER_URL`             | string | "https://ds-header.nypl.org"                                                                 | The base URL of the NYPL environment-specific header and footer scripts.                                     |
+| `ADOBE_EMBED_URL`             | string | ""                                                                                           | Url endpoint used for Adobe Analytics event tracking.                                                        |
+| `SHEP_API`                    | string | ""                                                                                           | SHEP API endpoint used for fetching Subject Heading data                                                     |
```

Comment:
Do you still need this? Doesn't look like it's used anymore, including the config in `apiEndpoints` in `src/config/config.ts`.

Reply:
If you don't need this, you can also remove it from `.env.example`.